test/src/test/java/org/corfudb/runtime/object/transactions/OptimisticTransactionContextTest.java

```diff
@@ -36,6 +39,60 @@ public void TXBegin() {
         OptimisticTXBegin();
     }
 
+    @Test
+    public void testSpanningTransaction() {
```

Comment:
Can you add some comments to this test which explains what it is trying to test?